src/www/vpn_ipsec_phase1.php

```diff
@@ -491,6 +494,9 @@ function ipsec_ikeid_next() {
                     }
                     break;
                 case 'hybrid_rsa_server':
+                    $(".auth_eap_tls").show();
```

Comment:
```suggestion
                    $('.auth_eap_tls').show();
```

Reply:
Hi @MichaelDeciso 
 I just copied from the code above, which also uses double quotes everywhere, to make it more consistent. Why should the quotes be changed?
Thanks

Reply:
It's what we do for all new code. The old code mostly uses double quotes, you're correct about that.

Reply:
okay